websites/M/Mahara-Tech/metadata.json

```diff
@@ -0,0 +1,40 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "605894319408283678",
+		"name": "eljooker"
+	},
+	"service": "Mahara-Tech",
+	"description": {
+		"en": "information technology learning platform"
+	},
+	"url": "maharatech.gov.eg",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/SjtqzAF.png",
```

Comment:
Logos must be at least [512x512px](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).